packages/suite/src/reducers/wallet/blockchainReducer.ts

```diff
@@ -131,6 +134,9 @@ const blockchainReducer = (state: BlockchainState = initialState, action: Action
                     },
                 };
                 break;
+            case BLOCKCHAIN.SYNCED:
+                draft[action.payload.symbol].syncTimeout = action.payload.timeout;
```

Comment:
since this timeout is here forever maybe we should catch `BLOCKCHAN.ERROR` to reset it?
example: removing/disabling coin > blockchain.disconnect > blockchain.error

Reply:
Added `clearTimeout` to blockchainActions.onDisconnect (which is called on `BLOCKCHAIN.ERROR`), therefore all sync `setTimeout`s and `clearTimeout`s are in the same file.
https://github.com/trezor/trezor-suite/pull/5249/commits/952819e2c925c5de94016d694ab42ae12418ebee